tests/functional/test_globals.py

```diff
@@ -0,0 +1,21 @@
+import os
+import unittest
+
+from awscli.testutils import create_clidriver
+from awscli.clidocs import EXAMPLES_DIR, GlobalOptionsDocumenter
+
+
+class TestGlobalOptionsDocumenter(unittest.TestCase):
+    def setUp(self):
+        self.driver = create_clidriver()
+        self.globals = GlobalOptionsDocumenter(self.driver)
+
+    def test_global_options(self):
+        filepath = os.path.join(EXAMPLES_DIR, 'global_options.rst')
+        self.assertEqual(self.globals.doc_global_options(),
+                         open(filepath).read())
```

Comment:
For all `open`s we should make sure to close the file handle to leak file descriptors. So instead we should let's just wrap these with a context manager e.g.:
```python
with open(filepath, 'r') as f:
     self.assertEqual(self.globals.doc_global_synopsis(), f.read())
```